dbos-config.schema.json

```diff
@@ -62,13 +59,11 @@
         "app_db_client": {
           "type": "string",
           "description": "Specify the database client to use to connect to the application database",
-          "enum": [
-            "pg-node",
-            "prisma",
-            "typeorm",
-            "knex",
-            "drizzle"
-          ]
+          "enum": ["pg-node", "prisma", "typeorm", "knex", "drizzle"]
```

Comment:
Is the schemas change necessary now that we're using `ORMEntities`? If not, let's remove it.

Reply:
Just removed it.